R/pkg/inst/worker/worker.R

```diff
@@ -36,7 +36,14 @@ compute <- function(mode, partition, serializer, deserializer, key,
       # available since R 3.2.4. So we set the global option here.
       oldOpt <- getOption("stringsAsFactors")
       options(stringsAsFactors = FALSE)
-      inputData <- do.call(rbind.data.frame, inputData)
+
+      # Handle binary data types
+      if ("raw" %in% sapply(inputData[[1]], class)) {
+        inputData <- SparkR:::rbindRaws(inputData)
```

Comment:
No it is not a preferred style in worker.R. It seems that they were some changes left slip under some previous code review. 
{code}
suppressPackageStartupMessages(library(SparkR))
{code}
should be moved to the front of work.R, and thus SparkR::: can be removed. A lot of "SparkR:::" is annoying.